calendar-adjustment/metadata.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Calendar adjustment for monthly time series",
+  "description": "Given a dataset containing monthly time series data with datestamps, normalizes the time series values by the number of days per month (with leap year detection). Suitable for monthly aggregated data (e.g. total liters per month).",
+  "kind": "script",
+  "source_code": "script.whizzml",
+  "inputs": [
+    {
+      "name": "id",
+      "type": "dataset-id",
+      "description": "The dataset that contains one or more datestamped monthly time series"
+    },
+    {
+      "name" : "data-fds",
+      "type" : "list",
+      "default" : [],
+      "description" : "List of field IDs which contain unadjusted monthly time series. Defaults to every numeric, parent field."
```

Comment:
another nit: "parent" is a bit of internal lingo... for users out there, i'd say "non-autogenerated"